RDF-Pipeline/t/tests/0047_Debug_Details/test-script

```diff
@@ -20,6 +20,7 @@ my $wwwDir = shift @ARGV || $ENV{RDF_PIPELINE_WWW_DIR} || die;
 $ENV{RDF_PIPELINE_WWW_DIR} = $wwwDir;	
 
 ############ CUSTOMIZE HERE ################
+warn "This test will normally fail, and that is okay.\n";
```

Comment:
Sorry to nit pick - it would be good to note why it's OK for this to fail and why we are keeping it in a comment if not the output.